### PR TITLE
re null values

### PR DESCRIPTION
**Pull request by sjvans**
hi @hjboth 

typo + i don't unterstand the example 😅

thanks,
sebastian

### CODE REVIEW

**Comment by renejeglinsky** [comment]
Hi @stewsk ,
what is your take here? (How) Should we change the example?

File: cds/cdl.md [hunk 1]
```diff
@@ -160,14 +160,14 @@ For arrayed types the `null` and `not null` constraints apply to the _members_ o
...
-In the following example the collection `emails` may hold members that are `null`. It may also hold a member where the element `kind` is `null`. The collection `email` must not be `null`!
+In the following example the collection `emails` may hold members that are `null`. It may also hold a member where the element `kind` is `null`. The collection `emails` must not be `null`!
 
 ```cds
 entity Bar {
     emails      : many {
         kind    : String null;
         address : String not null;
-    } null;
+    } null; // <- so the example shall show that this is ignored?!
```
[the remark this answers is not in the document]
Comment (renejeglinsky) [reply cds/cdl.md:170]: So I do understand, that the sample is correct. But does it need more explanation? Is it maybe not obvious that the final `null` is related to one member of `emails` can be null but not the whole collection of `emails`? (This is how I understood the discussion, of course that could contain misunderstandings)

[the remark this answers is not in the document]
Comment (stewsk) [reply cds/cdl.md:170]: Yes, I think the example is correct, but indeed needs a bit more explanation. I'm going to try to improve it, but first need to talk to @hjboth to make sure I don't make it worse :-)

[the remark this answers is not in the document]
Comment (agoerler) [reply cds/cdl.md:170]: > Yes, I think the example is correct, but indeed needs a bit more explanation. I'm going to try to improve it, but first need to talk to @hjboth to make sure I don't make it worse :-)

Isn't this already clear from the docu:

https://cap.cloud.sap/docs/cds/cdl#null-values

_For arrayed types the null and not null constraints apply to the members of the collections. The default is not null indicating that the collections can't hold null values.
_

Maybe we just need to extend this to to-many associations...

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply cds/cdl.md:170]: @stewsk Have you had the chance to work in this?

**Review by renejeglinsky — APPROVED** [review]

**Comment by renejeglinsky** [comment]
Thanks Steffen!

**Pull request merged**